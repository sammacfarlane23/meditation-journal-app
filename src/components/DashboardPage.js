import React from 'react';
import EntryList from './EntryList';
import Footer from './Footer';

export default () => (
  <div className='dashboard-layout'>
    <div className='content-container'>
      <EntryList />
    </div>
    <Footer />
  </div>
);
