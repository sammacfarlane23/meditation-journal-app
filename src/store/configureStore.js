import { createStore, combineReducers, applyMiddleware } from "redux";
import thunk from "redux-thunk";
import authReducer from "../reducers/auth";
import entriesReducer from "../reducers/entries";
import filtersReducer from "../reducers/filters";

const configuredStore = () => {
  const store = createStore(
    combineReducers({
      entries: entriesReducer,
      auth: authReducer,
      filters: filtersReducer,
    }),
    applyMiddleware(thunk)
  );

  return store;
};

export default configuredStore;
