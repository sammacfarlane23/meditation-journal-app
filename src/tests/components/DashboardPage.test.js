import React from 'react';
import DashboardPage from '../../components/DashboardPage';
import { renderWithState } from '../store/mockStore';

test('should render dashboard page correctly', () => {
  const component = renderWithState(<DashboardPage />);
  expect(component).toMatchSnapshot();
});
