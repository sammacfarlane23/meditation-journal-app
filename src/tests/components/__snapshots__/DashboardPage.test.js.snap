// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render dashboard page correctly 1`] = `
<div
  className="dashboard-layout"
>
  <div
    className="content-container"
  >
    <[object Object] />
  </div>
  <[object Object] />
</div>
`;
